canvas_modules/common-canvas/src/common-properties/controls/dropdown/dropdown.jsx

```diff
@@ -87,6 +89,20 @@ class DropDown extends React.Component {
 		return selectedOption;
 	}
 
+	getItemIcon(enumValue) {
```

Comment:
enumValue isn't passed into the function so this wouldn't be passed down.

Can you also add a test case to make sure this actually works correctly?  You should validate the propertId and enumValue are correct.  Might also be worth having 2 different icons in the example to show this works.